pkg/installmanager/installmanager.go

```diff
@@ -72,6 +72,7 @@ const (
 	sshCopyTempFile                     = "/tmp/ssh-privatekey"
 	defaultInstallConfigMountPath       = "/installconfig/install-config.yaml"
 	defaultManifestsMountPath           = "/manifests"
+	hiveSSHKnownHostsAnnotation         = "hive.openshift.io/ssh-known-host"
```

Comment:
This is pretty hacky and I don't think it's the permanent solution, we'll have libvirt in a pod eventually,I could rename the annotation to indicate it's not something you should consider permanent?

Reply:
We can leave the annotation name as is. I don't think there is any danger of someone using this and us dropping support for it.